test/service/CertificateService.test.ts

```diff
@@ -70,26 +70,9 @@ describe('CertificateService', () => {
         expect(expectedMetadata).deep.eq(BootstrapUtils.loadYaml(join(target, 'metadata.yml'), false));
 
         const files = await fsPromises.readdir(target);
-        expect(files).deep.eq([
-            'ca.cert.pem',
-            'ca.cnf',
-            'ca.pubkey.pem',
-            'index.txt',
-            'index.txt.attr',
-            'index.txt.attr.old',
-            'index.txt.old',
-            'metadata.yml',
-            'new_certs',
-            'node.cnf',
-            'node.crt.pem',
-            'node.csr.pem',
-            'node.full.crt.pem',
-            'node.key.pem',
-            'serial.dat',
-            'serial.dat.old',
-        ]);
+        expect(files).deep.eq(['ca.cert.pem', 'ca.pubkey.pem', 'metadata.yml', 'node.crt.pem', 'node.full.crt.pem', 'node.key.pem']);
```

Comment:
i assume metadata.yml was kept intentionally?

Reply:
Yes, metadata is mine. It holds the used main and node public keys. It's not easy nodejs to read the pem files without a library.